language/tools/move-unit-test/tests/test_sources/cross_module_aborts.evm.exp

```diff
@@ -7,7 +7,7 @@ Test failures:
 Failures in 0x1::B:
 
 ┌── failing_test ──────
-│ Test was not expected to abort but it aborted with 0 here
+│ Test was not expected to fail, but it aborted with code 0 originating in an unknown location. Rooted here
```

Comment:
The "Rooted here" at the end seems to dangle a bit strangely, can we make it part of the same sentence?
```suggestion
│ Test was not expected to fail, but it aborted with code 0 originating in an unknown location rooted here
```